### PR TITLE
Move branches into a separate interpreter group.

### PR DESCRIPTION
**Pull request by zherczeg**
The result processing of branches are removed (breaks are changed to continue) and branch is parsed as GET

Benchmark | Perf (sec) |
----: | ----: | 
3d-cube.js | 1.308 -> 1.272 : +2.822% | 
3d-raytrace.js | 1.538 -> 1.505 : +2.126% | 
access-binary-trees.js | 0.745 -> 0.740 : +0.662% | 
access-fannkuch.js | 3.122 -> 3.048 : +2.376% | 
access-nbody.js | 1.525 -> 1.515 : +0.665% | 
bitops-3bit-bits-in-byte.js | 0.751 -> 0.689 : +8.241% | 
bitops-bits-in-byte.js | 1.048 -> 1.005 : +4.106% | 
bitops-bitwise-and.js | 1.976 -> 1.907 : +3.500% | 
bitops-nsieve-bits.js | 2.474 -> 2.428 : +1.878% | 
controlflow-recursive.js | 0.542 -> 0.526 : +3.077% | 
crypto-aes.js | 1.448 -> 1.400 : +3.351% | 
crypto-md5.js | 0.899 -> 0.886 : +1.397% | 
crypto-sha1.js | 0.849 -> 0.834 : +1.844% | 
date-format-tofte.js | 1.146 -> 1.136 : +0.892% | 
date-format-xparb.js | 0.576 -> 0.578 : -0.296% | 
math-cordic.js | 1.887 -> 1.796 : +4.806% | 
math-partial-sums.js | 1.057 -> 1.042 : +1.405% | 
math-spectral-norm.js | 0.789 -> 0.760 : +3.726% | 
string-base64.js | 4.640 -> 4.650 : -0.234% | 
string-fasta.js | 2.187 -> 2.196 : -0.381% | 
Geometric mean: | +2.319% |

### CODE REVIEW

File: jerry-core/vm/vm.c [hunk 1]
```diff
@@ -755,25 +753,84 @@ vm_loop (vm_frame_ctx_t *frame_ctx_p) /**< frame context */
...
+      right_value = ecma_make_simple_value (ECMA_SIMPLE_VALUE_UNDEFINED);
+
+      uint32_t operands = VM_OC_GET_ARGS_INDEX (opcode_data);
+
+      if (operands >= VM_OC_GET_LITERAL)
+      {
+        uint16_t literal_index;
+        READ_LITERAL_INDEX (literal_index);
+        READ_LITERAL (literal_index,
+                      left_value);
```
Comment (LaszloLango) [inline jerry-core/vm/vm.c]: It's a short statement. Please write it in one line.

File: jerry-core/vm/vm.c [hunk 2]
```diff
@@ -755,25 +753,84 @@ vm_loop (vm_frame_ctx_t *frame_ctx_p) /**< frame context */
...
+
+        if (operands != VM_OC_GET_LITERAL)
+        {
+          switch (operands)
+          {
+            case VM_OC_GET_LITERAL_LITERAL:
+            {
+              uint16_t literal_index;
+              READ_LITERAL_INDEX (literal_index);
+              READ_LITERAL (literal_index,
```
Comment (LaszloLango) [inline jerry-core/vm/vm.c]: ditto

**Comment by LaszloLango** [comment]
LGTM

**Comment by dbatyai** [comment]
LGTM

**Pull request merged**